.git-blame-ignore-revs

```diff
@@ -0,0 +1,11 @@
+# This is a file that can be used by git-blame to ignore some revisions.
+# (git 2.23+, released in August 2019)
+#
+# Can be configured as follow:
+#
+#     $ git config blame.ignoreRevsFile .git-blame-ignore-revs
+#
+# For more information you can look at git-blame(1) man page.
+
+# Reformat yaml files
+df39695ac4c057fe254e7bd98eedb5ff98cacf43
```

Comment:
I think that this might be a wrong hash from a previous version of the tree

Reply:
Damn, good catch! Thanks!

Reply:
Fixed